docs/user/environment/0-setup/elevation.rst

```diff
@@ -0,0 +1,66 @@
+Elevation
+=========
+
+The elevation of the launch site of a rocket is particularly important for
+determining its interaction with the atmosphere and Earth's gravitational field.
```

Comment:
Does the pronoun "its" refer to the elevation or to the rocket? I believe its the latter, so we could possibly rewrite:
```suggestion
The elevation of the launch site is particularly important for
determining the rocket's interaction with the atmosphere and Earth's gravitational field.
```